public/app/core/services/keybindingSrv.ts

```diff
@@ -171,6 +171,24 @@ export class KeybindingSrv {
     this.bind(keyArg, withFocusedPanel(fn));
   }
 
+  setupTimeRangeBindings(updateUrl = true) {
+    this.bind('t z', () => {
+      appEvents.publish(new ZoomOutEvent({ scale: 2, updateUrl: updateUrl }));
```

Comment:
Nit: Can use property shorthand syntax
```suggestion
      appEvents.publish(new ZoomOutEvent({ scale: 2, updateUrl }));
```

Reply:
Sounds good!